src/components/chat-bubble/ChatBubble.scss

```diff
@@ -122,5 +122,7 @@
 	&-photo {
 		height: 30rem;
 		width: auto;
+		min-width: 100%;
+		object-fit: contain;
```

Comment:
`contain` will center the photo, `cover` will crop the photo, without this the photo will be stretched